hq/app/aws/support/TrustedAdvisorSGOpenPorts.scala

```diff
@@ -60,4 +61,10 @@ object TrustedAdvisorSGOpenPorts {
         }
     }
   }
+
+  def refreshSGOpenPorts(client: AWSSupportAsync)(implicit ec: ExecutionContext): Attempt[RefreshTrustedAdvisorCheckResult] = {
```

Comment:
If refresh is required here why is it not required for the other checks?

Reply:
I think it's actually required for other checks too. There is a refresh button next to each check on AWS Trusted Advisor -> Security page, so I imagine other checks needs refreshing too.